nanosynth.rb

```diff
@@ -44,17 +46,16 @@ def main
                                 # Amplitudes above 1.0 will result in distortion (or other weirdness).
 
   # Generate 1 second of sample data at the given frequency and amplitude.
-  # Since we are using a sample rate of 44,100Hz, 44,100 samples are required for one second of sound.
-  samples = generate_sample_data(wave_type, 44100, frequency, max_amplitude)
+  # Since we are using a specific sample rate measured in samples per second, that many samples are required for one second of sound.
+  samples = generate_sample_data(wave_type, SAMPLE_RATE * SECONDS_TO_GENERATE, frequency, max_amplitude)
```

Comment:
I like adding the `SAMPLE_RATE * SECONDS_TO_GENERATE` because to me it makes the relationship between the sample rate, the amount of time, and the number of samples to generate. Without multiplying by 👍